src/Workspaces/Core/Portable/Storage/SQLite/Interop/SqlConnection.cs

```diff
@@ -216,7 +229,7 @@ public Stream ReadBlob_MustRunInTransaction(string tableName, string columnName,
             }
 
             const int ReadOnlyFlags = 0;
-            var result = raw.sqlite3_blob_open(_handle, "main", tableName, columnName, rowId, ReadOnlyFlags, out var blob);
+            var result = raw.sqlite3_blob_open(_handle, dbName, tableName, columnName, rowId, ReadOnlyFlags, out var blob);
```

Comment:
@ericsink having problems here.  even though i attach to the same shared=cache memory db in each connection using: `ATTACH DATABASE 'file::memory:?cache=shared' AS writecache;`, and i then create all my tables in that db, i then fail here on the blob open here. This happens even after i successfully wrote the data using this statement:

`insert or replace into "writecache.DocumentData3"("DataId","Checksum","Data") values (?,?,?)`

the error i get here is:

```
raw.sqlite3_errmsg(_handle)
"no such table: writecache.DocumentData3"
```